content/en/docs/concepts/workloads/controllers/statefulset.md

```diff
@@ -280,6 +282,27 @@ updates to its `.spec.template` will not be propagated to its Pods.
 In most cases you will not need to use a partition, but they are useful if you want to stage an
 update, roll out a canary, or perform a phased roll out.
 
+### Maximum unavailable Pods
+
+{{< feature-state for_k8s_version="v1.24" state="alpha" >}}
+
+You can control the maximum number of Pods that can be unavailable during an update
+by specifying the `.spec.updateStrategy.rollingUpdate.maxUnavailable` field.
+The value can be an absolute number (for example, `5`) or a percentage of desired
+Pods (for example, `10%`). Absolute number is calculated from the percentage value
+by rounding it up. This field cannot be 0. The default setting is 1.
+
+This field applies to all Pods in the range `0` to `replicas-1`. If there is any
+unavailable Pod in the range `0` to `replicas-1`, it will be counted towards
+`maxUnavailable`.
```

Comment:
nit:
```suggestion
This field applies to all Pods in the range `0` to `replicas - 1`. If there is any
unavailable Pod in the range `0` to `replicas - 1`, it will be counted towards
`maxUnavailable`.
```

The space makes it more clear that we're using code style in place of equation style (which this site doesn't support).

Reply:
OK